### PR TITLE
Remove the content about vnc

### PR DESCRIPTION
**Pull request by mrFranklin**
Because of the cuttlefish has not support vnc, I removed the content about vnc.

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -267,84 +267,9 @@ Alternatively, you could run adb inside the container, and use the adb client ov
...
-The client will ask the VNC server address to connect, and this is the host ip and port to be used:
-
-```bash
-127.0.0.1:6444
-```
-
 ## WebRTC
 
-As an alternative to VNC, you can connect cuttlefish via webRTC. you can
+You can connect cuttlefish via webRTC. you can
```
Comment (jemoreira) [inline docker/README.md:272]: please also fix: "connect to cuttlefish"

**Review by jemoreira — COMMENTED** [review]
LGTM.

**Review by jemoreira — APPROVED** [review]

File: docker/README.md (hunk 1 above)
Comment (mrFranklin) [reply docker/README.md]: fixed

Comment (jemoreira) [reply docker/README.md]: Thanks!

**Pull request merged**